src/components/pages/DataHub/DataHub.js

```diff
@@ -207,9 +207,6 @@ const DataHub = props => {
         added, more data can be visualized.
       </h3>
       <h2 className="h1Styles">Approvals vs Denials Nationwide</h2>
-      <p className="divider">
```

Comment:
I see you've removed the 'divider' class from this component, are there any styles that you can remove as well. Maybe let's do a find-all in the app to see if we can't remove those styles altogether. 

This would assume that no other instances of the 'divider' styles would be referenced by the className attribute/prop. Hence a `find all` would be necessary to tackle both the removal of the divider and its associated styles. 

Thoughts?

Reply:
Did a quick find all for usages of the 'divider' class and they're all fine as is. The reason for this removal is that the designer preferred that the header text (Approvals vs Denials Nationwide) appear without a divider. The divider is necessary in all other instances.